### PR TITLE
Correct grammar, hyperlinks and formatting

### PR DESCRIPTION
**Pull request by ipadurean-bd**
Correct acronym defined first for each document
Use Alliance instead of CSA (two terms: Alliance and CSA were used for the Connectivity Standard Alliance)
Correct grammar mistakes
Correct the hyperlinks
Correct formatting
Apply uniform formatting of lists with dot at the end

### CODE REVIEW

**Comment by sammachin** [comment]
@ipadurean-bd thanks for the contribution,

We don't use the acronym CSA we prefer "The Alliance" as a shortened term. 
Other fixes look good, could you update your PR to remove the CSA changes

**Comment by ipadurean-bd** [comment]
@sammachin thank you!
I updated the PR so it contains the preferred shortened term

**Comment by sammachin** [comment]
@ipadurean-bd yeah I thought I'd fixed that yesterday, but your's is the first PR using it that's coming from an external repo so I can see its still failing. I'll take a look at that today,

**Comment by sammachin** [comment]
@ipadurean-bd 
I think I've found the issue, could you fetch the latest changes into your fork and it should then run, it was using the variables and secrets from your fork rather than the main repo

**Comment by ipadurean-bd** [comment]
@sammachin done

**Comment by sammachin** [comment]
seems like the action was still checking out the main branch not your head, I've just updated it so could you fetch that change and we'll see what it builds this time

**Comment by ipadurean-bd** [comment]
@sammachin what is the situation on this pull request? It cannot be merged?

**Comment by sammachin** [comment]
@ipadurean-bd could you pull in the latest changes from main, that should trigger the preview to be re-deployed, there were some bugs in the github action used for the previews but it was also running the action in main and showing that as the preview without your changes!
I also commented above that the change to /static for images is wrong, the handbook isn't meant to be read from github this is just the source for the static site generated with hugo and published at https://handbook.buildwithmatter.com

**Comment by ipadurean-bd** [comment]
@sammachin  so I should revert the images changes?

**Comment by sammachin** [comment]
> @sammachin so I should revert the images changes?

Yes please, as you can now see in the preview the images don't load https://d1teub3jrrfss6.cloudfront.net/pr-17/howitworks/datamodel/

**Comment by ipadurean-bd** [comment]
@sammachin I reverted the hyperlinks, but the preview still doesn't load the images

**Comment by sammachin** [comment]
yes it was the caching, I'll need to add an invalidation to the action in future,

Thanks I'll review the rest of the edits today

File: content/Alliance/_index.md [hunk 1]
```diff
@@ -5,12 +5,10 @@ weight = 2
 pre = "<b>2. </b>"
 +++
 
-
-# The Connectivity Standards Alliance
+# The Connectivity Standards Alliance (The Alliance)
```
Comment (sammachin) [inline content/Alliance/_index.md:8]:
```suggestion
# The Connectivity Standards Alliance
```
I agree that its useful to explain acroynms but this makes the H1 on the page far to long

**Review by sammachin — COMMENTED** [review]
One small edit but otherwise looks good

File: content/Alliance/_index.md (hunk 1 above)
Comment (ipadurean-bd) [reply content/Alliance/_index.md]: Done

**Pull request merged**